mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -165,6 +168,32 @@ def __init__(self, name=None, bus=None, use_settings=True):
         # Skill Public API
         self.public_api = {}
 
+        if bus:
```

Comment:
What happens if the bus is not passed in?

Reply:
This is to handle the current (v2) case which does not pass the bus. The bind, etc will need to be called externally after init.